Description: nfsiostat: replace 'list' reserved word
 list is a reserved word in python and should not be used as a variable name. 
 Changing list to devicelist for list_nfs_mounts().
Forwarded: https://marc.info/?l=linux-nfs&m=155346326413894&w=2
Applied-Upstream: http://git.linux-nfs.org/?p=steved/nfs-utils.git;a=commitdiff;h=c4c14011b70375050d7bba7c57e2eaf4c715dc7c
Bug-Debian: http://bugs.debian.org/925943
Bug-Ubuntu: https://launchpad.net/bugs/1821261
Author: Matthew Ruffell <matthew.ruffell@canonical.com>
Last-Update: 2019-04-02
Index: nfs-utils-1.3.4/tools/nfs-iostat/nfs-iostat.py
===================================================================
--- nfs-utils-1.3.4.orig/tools/nfs-iostat/nfs-iostat.py
+++ nfs-utils-1.3.4/tools/nfs-iostat/nfs-iostat.py
@@ -484,20 +484,20 @@ def list_nfs_mounts(givenlist, mountstat
        return a full list if the given list is empty -
        may return an empty list if none found
     """
-    list = []
+    devicelist = []
     if len(givenlist) > 0:
         for device in givenlist:
             stats = DeviceData()
             stats.parse_stats(mountstats[device])
             if stats.is_nfs_mountpoint():
-                list += [device]
+                devicelist += [device]
     else:
         for device, descr in list(mountstats.items()):
             stats = DeviceData()
             stats.parse_stats(descr)
             if stats.is_nfs_mountpoint():
-                list += [device]
-    return list
+                devicelist += [device]
+    return devicelist
 
 def iostat_command(name):
     """iostat-like command for NFS mount points
